cli/endpoints/batch/deploy-models/huggingface-text-summarization/deployment_torch113.yaml

```diff
@@ -2,8 +2,9 @@ $schema: https://azuremlschemas.azureedge.net/latest/batchDeployment.schema.json
 endpoint_name: text-summarization-batch
 name: text-summarization-hfbart
 description: A text summarization deployment implemented with HuggingFace and BART architecture
+type: model
 model: azureml:bart-text-summarization@latest
-compute: azureml:gpu-cluster-k80
+compute: gpu-cluster-k80
```

Comment:
Does this require GPU? It's infamously difficult to get GPU compute

Reply:
It does. This runs an LLM (BART). It would take hours otherwise.

Reply:
We have an article now that explains [how to run LLMs on batch](https://learn.microsoft.com/en-us/azure/machine-learning/how-to-nlp-processing-batch?view=azureml-api-2&tabs=cli).